### ISSUE TITLE
Allow results to be written to a file

### ISSUE THREAD

**Issue by akielthy** [powerus117/RLCGPerformanceTracker#3]
I have changes locally to save each run to a timestamped json file.
Can I get permissions to create branches/PRs so I can push

![image](https://github.com/user-attachments/assets/76d4eaf1-2777-4a7e-a77c-edad98aa8dc5)
![image](https://github.com/user-attachments/assets/a42c85a8-0a61-4f52-a19b-d877856e0857)

**Comment by dschwick** [comment]
Would you be able to describe how you did it locally? I'd like to do this with mine as well but am no where near competent enough with writing code

**Comment by akielthy** [comment]
> Would you be able to describe how you did it locally? I'd like to do this with mine as well but am no where near competent enough with writing code

Absolutely, I am away at the moment but will get it to you in a few days. You can launch runelite with the modified plugin but I remember it being a bit bare bones, it would be best to have the changed merged here so they can be used from the plugin hub. I'll see if I can contact anyone to make it happen

**Comment by akielthy** [comment]
> Would you be able to describe how you did it locally? I'd like to do this with mine as well but am no where near competent enough with writing code

I have committed my changed to a fork on my profile, also created a PR to bring them back to the main plugin. Output files will be in  .runelite/gauntletPerformanceTracker
